ksql-engine/src/main/java/io/confluent/ksql/function/udf/json/ArrayContainsKudf.java

```diff
@@ -75,7 +76,10 @@ public Object evaluate(final Object... args) {
       return jsonStringArrayContains(searchValue, (String) args[0]);
     } else if (args[0] instanceof Object[]) {
       return ArrayUtil.containsValue(searchValue, (Object[]) args[0]);
+    } else if (args[0] instanceof Collection) {
```

Comment:
This should always be either a List or a String

Reply:
I removed the Object[] condition in a new patch. I verified that JSON and AVRO formats can still use ArrayContains without issues.

Reply:
Btw, I added tests on QueryTranslationTest to verify ARRAYCONTAINS works as expected and it is not broken again.